packages/tasks/src/tasks/image-feature-extraction/about.md

```diff
@@ -0,0 +1,24 @@
+## Use Cases
+
+### Transfer Learning
+
+Models trained on a specific dataset can learn very shallow features about the data. For instance, a model trained on car classification dataset learns to recognize edges and curves on a very high level and car-specific features on a low level. This information can be transferred to a new model that is going to be trained on classifying trucks. This way of extracting features and transferring to a model is called feature extraction. One can pass their dataset through a feature extraction pipeline and feed the result to a classifier directly.
```

Comment:
```suggestion
Feature extraction is a process that transforms raw input data into numerical representations that try to preserve the information in the original dataset. Machine learning models trained on a dataset learn distinctive features about the data and can be used as feature extractors. For instance, a model trained on a car classification dataset may automatically learn to recognize edges and curves on a very high level, and car-specific features such as wheels or chassis shapes. The extracted features can be used to train downstream tasks, such as a truck classifier.
```

I personally find a bit confusing the identification of feature extraction with transfer learning.

Reply:
in the context of hugging face models it's mostly used for that no? a generic definition would confuse users on what they should do with hugging face models.

Reply:
> the [definition](https://github.com/huggingface/huggingface.js/pull/559/files/afc9a1f7e8e9b7a92595c426847688b857f1dc17#diff-97656c9eaf51ebd19b884b88d76afb46453747a0c18abc9b6348573d5b622667R48) in the data.ts doesn't really identify with transfer learning

I was referring to the sentence you wrote "This way of extracting features and transferring to a model is called feature extraction", I don't think it's accurate or I'm not understanding it right. How about the following then?

```suggestion
Models trained on a specific dataset can learn features about the data. For instance, a model trained on a car classification dataset learns to recognize edges and curves on a very high level and car-specific features on a low level. This information can be transferred to a new model that is going to be trained on classifying trucks. This process of extracting features and transferring to another model is called transfer learning.
```

If you think it's right, then we can similarly adapt the paragraph in the general feature extraction task.

Reply:
Any thoughts about this? It basically rephrases "this way of extracting features and transferring to a model is called feature extraction", which I find confusing.